src/containers/other/NFTView/NFTView.tsx

```diff
@@ -411,10 +409,11 @@ const NFTView: React.FC<NFTViewProps> = ({ onTransfer }) => {
 		return (
 			<>
 				<div className={styles.Stats}>
-					{data.map((item) => (
+					{data.map((item, index) => (
 						<>
 							{!item.isHidden ? (
 								<StatsWidget
+									key={index}
```

Comment:
Use something other `index` as the `key` here - [iirc it's bad practice to use index](https://stackoverflow.com/questions/46735483/error-do-not-use-array-index-in-keys). I don't think we would run into problems (currently) by using `index` here, but best to change it in my opinion.

Reply:
Agreed! Just updated the key so that we won't have redundant key value. Thanks